astropy/io/fits/column.py

```diff
@@ -2502,18 +2502,14 @@ def _dtype_to_recformat(dtype):
     Utility function for converting a dtype object or string that instantiates
     a dtype (e.g. 'float32') into one of the two character Numpy format codes
     that have been traditionally used by Astropy.
-
-    In particular, use of 'a' to refer to character data is long since
```

Comment:
@pllim - it was this comment that made me think it is totally fine to just replace, without even a changelog message - essentially, the "a" dtype was never seen by users. Even on older numpy,
```
In [34]: np.dtype('a1')
Out[34]: dtype('S1')
```

Reply:
Ah, okay!